.github/workflows/ci.yml

```diff
@@ -26,4 +26,17 @@ jobs:
         docker run -d -p 50051:50051 casbin/casbin-server
       
     - name: Go Test
-      run: go test ./...
\ No newline at end of file
+      run: go test ./...
+
+  semantic-release:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - name: Run semantic-release
+        if: github.repository == 'casbin/casbin' && github.event_name == 'push'
```

Comment:
`github.repository == 'casbin/casbin-go-client'`

Reply:
We can also create a  workflow for release this.

Reply:
@sagilio Done. Thanks :)
@nodece Wouldn't the action already release this? I'm sorry, I am a bit confused. Thanks

Reply:
@nodece